bittide-extra/src/Protocols/Wishbone/Extra.hs

```diff
@@ -0,0 +1,53 @@
+-- SPDX-FileCopyrightText: 2025 Google LLC
+--
+-- SPDX-License-Identifier: Apache-2.0
+
+module Protocols.Wishbone.Extra (delayWishboneC) where
+
+import Clash.Prelude
+import Protocols
+import Protocols.Wishbone
+
+data DelayWishboneState aw n a
+  = WaitingForManager
+  | WaitingForSubordinate (WishboneM2S aw n a)
+  | AcknowledgingManager (WishboneS2M a)
+  deriving (Generic, NFDataX)
+
+{- | Breaks the combinatorial path between a Wishbone manager and subordinate by inserting
+a Moore machine.
+-}
+delayWishboneC ::
```

Comment:
Please mention that it adds two cycles of delay to each transaction. One to capture the request and one to capture the response